pulpcore/cli/rpm/remote.py

```diff
@@ -44,6 +47,12 @@ def remote(ctx: click.Context, pulp_ctx: PulpContext, remote_type: str) -> None:
         "--policy", type=click.Choice(["immediate", "on_demand", "streamed"], case_sensitive=False)
     ),
     click.option("--sles-auth-token"),
```

Comment:
I think this option needs to be scoped for the other context now.

Reply:
Changed it to pulp_option to allow it with context.